network-primitives/bootstrap_node_provisioner.tf

```diff
@@ -119,6 +119,8 @@ resource "null_resource" "start-boostrap-nodes" {
       "sudo chmod +x /subspace/create_compose_file.sh",
       "sudo /subspace/create_compose_file.sh ${var.bootstrap-node-config.reserved-only} ${length(local.bootstrap_nodes_ip_v4)} ${count.index}",
       "docker compose -f /subspace/docker-compose.yml up -d --remove-orphans",
+      "sudo mkdir -p /etc/iptables/",
+      "sudo iptables-save > /etc/iptables/rules.v4",
```

Comment:
Hm... this will store Docker rules, I'd say we need to avoid it if at all possible, it will almost certainly become a source of problems at some point. Can you explicitly write just those that are necessary to block private networks?

Reply:
no, this will overwrite the docker rules I believe. In this case, the is fine because we are not going to delete the docker networks. even if we did, the docker created its own iptable chains so i believe it should do a full replace within those chains.

Reply:
I mean it will store whatever there is in iptables right now, which is not even guaranteed to be the same on all machines. If something changes in next Docker upgrade or whatever, it'll break. Do we need more than one line stored there? If not then `echo "..." > ` would be much more reliable.